oe_corporate_site_info.module

```diff
@@ -59,23 +60,28 @@ function oe_corporate_site_info_form_system_site_information_settings_alter(&$fo
   }
 
   $form['oe_site_information']['content_owners'] = [
+    '#type' => 'multivalue',
+    '#cardinality' => MultiValue::CARDINALITY_UNLIMITED,
```

Comment:
This can be skipped, it defaults to unlimited, so we don't need to import the class neither.